File: PEMROGRAMAN 1/PRAKTIKUM MODUL 3/Bahasa Python/PRAK302_2410817310009_Muhamad_Ghazi_Rakhmadi.py
nilai = int(input("Input: "))

if nilai >= 80:
    huruf = "A"
elif nilai >= 70:
    huruf = "B"
elif nilai >= 60:
    huruf = "C"
elif nilai >= 50:
    huruf = "D"
else:
    huruf = "E"

print(f"Ouput: {huruf}")

File: PEMROGRAMAN 1/PRAKTIKUM MODUL 3/Bahasa C/PRAK302_2410817310009_Muhamad_Ghazi_Rakhmadi.c
#include <stdio.h>
int main () {
    int nilai;
    char grade;

    printf("Input: ");
    scanf("%d", &nilai);

    if (nilai >= 80) {
        grade = 'A';
    } else if (nilai >= 70) {
        grade = 'B';
    } else if (nilai >= 60) {
        grade = 'C';
    } else if (nilai >= 50) {
        grade = 'D';
    } else {
        grade = 'E';
    }
   
    printf("Output: %c\n", grade);

    return 0;
}